test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -1254,7 +1254,7 @@ def test_server_restart2(self):
         a = {'resources_used.foo_i': 29,
              'resources_used.foo_f': 0.29,
              'resources_used.foo_str':
-             "\'{\"eight\": 8, \"seven\": 7, \"nine\": 9}\'"}
+             "\'{\"eight\": 8, \"nine\": 9, \"seven\": 7}\'"}
```

Comment:
why did you change the order? was the test failing?

Reply:
Yes.

Reply:
Will revert this change and file a separate ticket to handle this test failure/bug.